plugins/colorsource/export.js

```diff
@@ -0,0 +1,332 @@
+// see https://github.com/standard-things/esm#getting-started
+require = require(`esm`)(module); // eslint-disable-line no-global-assign
+const uuidV5 = require(`uuid/v5`);
+
+const {
+  CoarseChannel,
+  FineChannel,
+  SwitchingChannel
+} = require(`../../lib/model.js`);
+const { scaleDmxValue } = require(`../../lib/scale-dmx-values.js`);
+
+module.exports.name = `ColorSource`;
+module.exports.version = `0.1.0`;
+
+const EDITOR_VERSION = `1.1.1.9.0.4`;
+
+const CHANNEL_TYPE_NO_FUNCTION = 0;
+const CHANNEL_TYPE_INTENSITY = 1;
+const CHANNEL_TYPE_POSITION = 2;
+const CHANNEL_TYPE_MULTI_COLOR = 3;
+const CHANNEL_TYPE_BEAM = 4;
+const CHANNEL_TYPE_COLOR = 5;
+
+const UUID_NAMESPACE = `0de81b51-02b2-45e3-b53c-578f9eb31b77`;
```

Comment:
Is this taken from somewhere or is it just a random seed?

Reply:
I admit I don't know. But it shouldn't matter, anyway.